README.rst

```diff
@@ -80,17 +80,19 @@ Example local run
   WARNING:tensorflow:From /home/fpokorny/.local/share/virtualenvs/tensorflow-FigIdZQa/lib/python3.6/site-packages/tensorflow/python/framework/op_def_library.py:263: colocate_with (from tensorflow.python.framework.ops) is deprecated and will be removed in a future version.
   Instructions for updating:
   Colocations handled automatically by placer.
-  512 x 512 matmul took:   	27366.3938 ms,	 0.01 GFLOPS
+  512 x 512 matmul took:   	38992.1272 ms,	 0.01 GFLOPS
```

Comment:
It should be quite different, using median instead of sum, something in the order of ```512 x 512 matmul took:          1.0850 ms,       247.15 GFLOPS```

Reply:
Yes, it should be something close to this number devided by number of iterations. I screwed rebase, sorry for that. Now it should be updated.

Reply:
No problems :) Thanks @fridex :+1: